theme/styles/snippets/product-grid-item.scss

```diff
@@ -0,0 +1,43 @@
+.ProductGridItem {
+  display: grid;
+
+  //Aspect ratio 
+  &::before {
+    content: "";
+    display: block;
+    height: 0;
+    width: 0;
+    padding-bottom: calc(480/499 * 100%);
+    grid-area: 1 / 1 / 2 / 2;
+  }
```

Comment:
The site doesn't have a max width so the content should stretch but maintain aspect ratio.